packages/webview_flutter/webview_flutter/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-## NEXT
+## 3.0.1
 
 * Removes a duplicate Android-specific integration test.
 * Fixes an integration test race condition.
+* Fix comments (accidentially mixed // with ///).
```

Comment:
"Fixes". (Per the style guide linked from the PR checklist.)

Reply:
Done